src/lib.rs

```diff
@@ -401,7 +401,10 @@ pub fn get_mime_extensions_str(mut mime_str: &str) -> Option<&'static [&'static
     }
 
     let (top, sub) = {
-        let split_idx = mime_str.find('/').unwrap();
+        let split_idx = match mime_str.find('/') {
+            Some(idx) => idx,
+            None => return None,
+        };
```

Comment:
You can just use `?` instead of the `match` here.

Reply:
You made my day, I did not know that early return worked on `Option` too.